docs/siem/detections/alerts-ui-manage.asciidoc

```diff
@@ -0,0 +1,98 @@
+[[alerts-ui-manage]]
+[role="xpack"]
+== Managing detection alerts
+
+The Detections page displays all alerts. From the Alerts table, you can change
```

Comment:
Not sure if it makes sense to clarify here since it's a sub-page (and surmisable from the details below), but you can no longer view `external alerts` on the Detections page unless you have the `External Alerts` rule enabled. I only say this as we've removed the dedicated `External Alerts` tab, and am not sure how our language of `all alerts` translates throughout the docs. Feel free to mark as resolved if you think this is fine, I  am heavily biased from being so close these last few months 😅

Reply:
@spong - edited slightly - I think it's ok now.